src/components/navigation/nav/index.js

```diff
@@ -23,6 +23,10 @@ const Nav = () => {
       noScroll.off()
     }
   })
+
+  const baseMetaDescription =
```

Comment:
Could pop these in as small functions to avoid having to string manipulate every time and have duplicated logic:

```js
const generateTitle = prefix => `${prefix} | Pride in London`;

...

description: generateTitle('Our story');
```